packages/react-components/react-icons-compat/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export { getWindow } from './getWindow';
+export { GlobalSettings } from './GlobalSettings';
+export type { IChangeEventCallback, IChangeDescription } from './GlobalSettings';
+export { setWarningCallback, warn } from './warn';
+export type { ISettingsMap } from './warn';
```

Comment:
I would suggest only exporting things explicitly related to the `icon` utilities for now to be as constrained as possible at the beginning, while keeping everything else needed internal to the package for now.

```suggestion
```